src/animation/AnimationAction.js

```diff
@@ -473,7 +473,7 @@ AnimationAction.prototype = {
 			if ( loopCount === -1 ) {
 				// just started
 
-				this.loopCount = 0;
+				this._loopCount = 0;
```

Comment:
Yes, looks like a bugfix. Currently, in case of `LoopOnce`, the actual loop count will not be updated to `0` as I originally intended.